### ISSUE TITLE
Cannot install - white screen (501 error)

### ISSUE THREAD

**Issue by Quin452** [Open-Web-Analytics/Open-Web-Analytics#824]
**Before you open an issue:**

- [x] I followed the [installation](https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/Installation) or [update](https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/Updating) guide
- [x] [I read the troubleshooting guide in the wiki](https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/Troubleshooting) 
- [] I have attached http headers, console/shell errors, or debug from OWA's error log file.

**OWA Version**: 1.7.7

**PHP Version**: 7.4.3

**Webserver**: Apache

**Browser**: Firefox Developer Edition 98.0b8

**Description**  
I install a fresh LAMP stack on a virtual machine.  I can load a static index.html page, but whenever I try OWA's index.php or install.php, I get a white screen and 501 error in Dev Tools; this happens when using a subdomain or IP address.
The permissions have been set to 755 and www-data.

**How to reproduce**  
Clean guest machine on a hypervisor, running Ubuntu 20.04 on both host and guest.
Nginx reverse proxy and Bind9 (for hairpin NAT).

**Possible Solution**  

**Additional context**

**Comment by jrbranaa** [comment]
I ran into this error as well.  I downloaded 1.7.6 and it seems to work.

**Comment by Quin452** [comment]
> I ran into this error as well. I downloaded 1.7.6 and it seems to work.

Did you use the zip file?
I've tried 1.7.6, and it is still the same issue (maybe caching?).

I wget the 1.7.6 zip, and extract it to a /owa directory (the files are in there, and not the [OWA-1.7.6] directory).
Permissions are 755 and www-data.

**Comment by Quin452** [comment]
Okay, so I removed the latest versions of PHP (7.4._something_) and MySQL (?).  I generally stick with just the "install PHP" and "install MySQL" commands without the versions.

[Following this guide](https://www.rosehosting.com/blog/how-to-install-open-web-analytics-on-ubuntu-18-04/), I was able to install 1.6.9, as 1.6.2 didn't work (I simply switch things out).
I am not sure at what point the installation breaks, but I did notice the extra fields.

Perhaps I'll have better luck upgrading it.
